### DISCUSSION TITLE
Set deploy status on pull request

### DISCUSSION THREAD

**Discussion by privatenumber** [JamesIves/github-pages-deploy-action#1061] [Ideas]
When using this action on a PR hook, I would like for it to update the banner in the PR to say that it has been deployed.

Currently, it says "This branch has not been deployed":
<img width="1266" src="https://user-images.githubusercontent.com/1075694/144704631-05a19e73-3559-42a8-8096-b3c23dba864c.png">

Seems like the deploy status can be set by sending a deployment event:
https://docs.github.com/en/rest/reference/repos#deployments

For reference, [nwtgck/actions-netlify](https://github.com/nwtgck/actions-netlify) does this:
https://github.com/nwtgck/actions-netlify/blob/bda6b2e6da6fdf666e0accd8873922698c37974d/src/main.ts#L42

**Comment by JamesIves** [comment]
I like the idea of this. Would this be something you're willing to open a PR for?
